runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerFlickOverlay.java

```diff
@@ -63,6 +63,12 @@ public Dimension render(Graphics2D graphics)
 			return null;
 		}
 
+		// Prayer orb widget will be hidden when minimap is hidden
+		Widget prayerOrb = client.getWidget(WidgetInfo.MINIMAP_PRAYER_ORB);
+		if (prayerOrb == null || prayerOrb.isHidden()) {
+			return null;
+		}
+
 		Widget xpOrb = client.getWidget(WidgetInfo.MINIMAP_QUICK_PRAYER_ORB);
 		if (xpOrb == null)
```

Comment:
you can do ` || xpOrb.isHidden()` then you don't need the extra widget check

Reply:
Thank you for your suggestion. I tested your code and it works fine for me. I have updated the PR.